mantidproject/mantid
# HistogramData Rollout FrameWork/Algorithms/ScaleX

### Description of work.

Refactored ScaleX.cpp and ScaleXTest.h to use HistogramData.
Added Performance tests in ScaleXTest.h
### To test:



Run ScaleX Algorithm on a workspace, check that the output is correct, functionality should not be changed.

Fixes #16876 



_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
